### ISSUE TITLE
mediaaccesspermission: not supported.

### ISSUE THREAD

**Issue by hidingfromu** [xMNG/Microsoft-Rewards-Bot#58]
Hi,

I'm receiving quite a lot of errors in the console. I'm currently using the dev version, but these errors occur on the main fork too. I'm based in the UK which probably makes a difference.
The most common error is

"[0507/095534.601:ERROR:web_contents_delegate.cc(197)] WebContentsDelegate::Check
MediaAccessPermission: Not supported."

This repeats constantly, on both forks, though the searches still complete.

"[0507/095827.195:INFO:CONSOLE(0)] "Access to font at 'https://i.s-microsoft.com/
fonts/segoe-ui/west-european/bold/latest.woff' from origin 'https://account.micr
osoft.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' hea
der is present on the requested resource.", source: https://account.microsoft.co
m/rewards/redeem/000702000242 (0)"

is a new one, as is:

"[0507/095922.980:INFO:CONSOLE(0)] "Mixed Content: The page at 'https://account.m
icrosoft.com/?lang=en-US&refd=account.live.com&refp=landing&mkt=EN-US' was loade
d over HTTPS, but requested an insecure image 'http://img-prod-cms-rt-microsoft-
com.akamaized.net/cms/api/am/imageFileData/RE1yq9a?w=150&h=150&q=60&m=6&o=f'. Th
is content should also be served over HTTPS.", source: https://account.microsoft
.com/?lang=en-US&refd=account.live.com&refp=landing&mkt=EN-US (0)"

and:

"[0507/100628.342:INFO:CONSOLE(26)] "Synchronous XMLHttpRequest on the main threa
d is deprecated because of its detrimental effects to the end user's experience.
 For more help, check https://xhr.spec.whatwg.org/.", source: https://www.bing.c
om/search?q=Ferenc+Puskas&filters=BTQI%3A%221%22+BTCI%3A%221%22+ShowTimesTaskPan
eTrigger%3A%22false%22&skipopalnative=true&rnoreward=1&FORM=ML151V&BTCA=1&BTSPC=
0 (26)"

I'll try the main fork tomorrow and update with any other errors that appear.

**Comment by hidingfromu** [comment]
"[0507/100744.954:INFO:CONSOLE(24)] "[MARKUP VALIDATION] Duplicate/blank htmlids:
 "List-of" (2 occurrences)", source: https://www.msn.com/en-gb/lifestyle/horosco
pes?ocid=MICROSOFTREWARDS&rnoreward=1 (24)"

Just popped :)

**Comment by ShoGinn** [comment]
Seems like a there are components of the webpage that Chromedriver is throwing errors.

Not much we can do to fix these

**Comment by hidingfromu** [comment]
> Seems like a there are components of the webpage that Chromedriver is throwing errors.
> 
> Not much we can do to fix these

Hi @ShoGinn, firstly thank you for your great work on this project. It's very handy and has actually gotten me interested in Python myself (don't worry I don't want to be your apprentice :P).

The second two errors only seem to occur using the dev build. I swapped back to the main build and only the error referenced in the title occurred. The dev build also failed on 3 of my accounts, whereas the main build succeeds on all accounts even though it throws errors all over the place.

Nothing needs doing about it, as it all works eventually, just thought I would put these errors up for reference. I'll also put up any solutions I find, not in an annoying way though :P

**Comment by ShoGinn** [comment]
@hidingfromu thats interesting! I am curious what chrome preferences are causing those errors to pop up......

**Comment by ShoGinn** [comment]
Just to follow up, what options are you using?
That way I can check to see what chrome options are throwing the codes.

**Comment by hidingfromu** [comment]
@ShoGinn Chrome options? Everything is on default I beleive. I'm on version Version 74.0.3729.131 (Official Build) (32-bit), Windows 7, but I get the same error on the x64 version of chrome on Windows 10.

**Comment by ShoGinn** [comment]
how are you running the script? Headless? --all --mobile --quiz??

**Comment by hidingfromu** [comment]
python ms_rewards.py --headless --mobile --pc --quiz

**Comment by ShoGinn** [comment]
very odd.. I think I'll work on creating a way for the script to help debugging these problems so that we can create a standard to work from in everyone's use scenarios.

Not much I can do now as it seems like everything is working.

Obviously with the latest pull if you were running any incorrect versions the script would stop you.

Give me until this weekend and I'll see what I can come up with.

**Comment by andrewjmetzger** [comment]
Please close issue @xMNG. Not only is status `wontfix`, the user account was deleted.
